### PR TITLE
File name and Robot test case refactoring

### PR DESCRIPTION
**Pull request by mrrajan**
This PR includes refactoring of File naming and test cases to include,
      - File name to have all lower case 
      - Separators  `_` for file naming
      - File renaming to have extension YAML
      - Test cases and keywords to use Title Case
      - Object locator variable to follow naming OR naming convention 
   
Signed-off-by: Rajan Ravi <rravi@redhat.com>

### CODE REVIEW

**Comment by mrrajan** [comment]
Renamed the directories with lowercases and renamed the tests directory to temp (This would be removed/refactored on upcoming PR's)

File: resources/keywords/login.resource [hunk 1]
```diff
@@ -1,10 +1,10 @@
 *** Settings ***
 Library     SeleniumLibrary
 Library     BuiltIn
-Resource    ../OR/Login_OR.resource
+Resource    ../OR/login_obj.resource
```
Comment (harshkumarRH) [inline resources/keywords/login.resource:4]: It should be _objectrepo_ instead of _OR_

File: resources/keywords/navigation.resource [hunk 2]
```diff
@@ -0,0 +1,26 @@
+*** Settings ***
+Library    String
+Resource    ../OR/login_obj.resource
+Resource    ../Keywords/login.resource
```
Comment (harshkumarRH) [inline resources/keywords/navigation.resource]: Can we just have ../login.resource

**Comment by harshkumarRH** [comment]
@mrrajan Looks good to me.
Only concern - Could you please change _OR_ to _objectrepo_ and _Keywords_ to _keywords_ where ever applicable

**Comment by mrrajan** [comment]
[result.zip](https://github.com/RHODA-lab/rhoda-ci/files/8574699/result.zip)
@psathyan Attached execution results for reference

**Review by harshkumarRH — APPROVED** [review]

**Review by ikanias — APPROVED** [review]
LGTM

**Pull request merged**